<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilteredView</class>
 <widget class="QWidget" name="FilteredView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilteredView</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="filterLayout" stretch="1,0,0">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>TEMP</string>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="buttonReload">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources/resources.qrc">
         <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LineFilter" name="filterString">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="viewWidget" native="true"/>
   </item>
  </layout>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Invalidate and load the current data.</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineFilter</class>
   <extends>QLineEdit</extends>
   <header>ui.linefilter</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonReload</sender>
   <signal>clicked()</signal>
   <receiver>actionReload</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
